boot/mynewt/syscfg.yml

```diff
@@ -28,6 +28,9 @@ syscfg.defs:
     BOOT_CUSTOM_START:
         description: 'Override hal_system_start with a custom start routine'
         value: 0
+    BOOT_PREBOOT:
+        description: 'Call boot_preboot() function before booting application'
+        value:
```

Comment:
What is the default supposed to be?

Reply:
For now by default function does not exist.
If package dfu from mynewt core will be included. It will provide this function, then function will check selected pin to see if DFU should be enabled and if so it will start TinyUSB/DFU functionality.
This will be very helpful for NRF53840 dongle where there is no debugger on board and flashing is requires special cable. Once this is enabled inserting dongle with BUTTON_1 pressed will start DFU and dfu-util or dfu-tool can be used to flash image into SLOT0.

But maybe someone will have other purpose for this so that is why I did not propose function name that implies DFU.

Reply:
No, I just meant, do you want to make the value `0` or `1`? Make it explicit.

Reply:
Right now I have dfu package that sets this to 1. This package is not BSP/APP/TARGET so it would not be able to set this value if it was already set.
If you want I can set this here and remove it there and add restriction that will force user to specify it in target.
Or I could add specific syscfg value that will include specific package from core and it will also set BOOT_PREBOOT on mcuboot side.

Reply:
> If you want I can set this here and remove it there and add restriction that will force user to specify it in target.
Or I could add specific syscfg value that will include specific package from core and it will also set BOOT_PREBOOT on mcuboot side.

Too much, thanks.